packages/validator/src/metrics.ts

```diff
@@ -206,6 +206,11 @@ export function getMetrics(register: MetricsRegister, gitData: LodestarGitData)
       help: "Total count of instances the attester duties dependant root changed",
     }),
 
+    attesterDutiesNextSlot: register.gauge({
+      name: "vc_attestation_duty_slot",
```

Comment:
you might mention in a comment that the naming here is important

Reply:
Yeah, it's probably a good idea to add some context to the metric and how it is used since we are not referencing it anywhere in our dashboards yet. This could be a good addition to the Lodestar Summary dashboard but I haven't found a good place for the panel yet.